browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -230,6 +237,34 @@ class MarkdownNoteDetail extends React.Component {
     }
   }
 
+  handleFullScreenButton (e) {
+
+    this.state.fullScreen.status = !this.state.fullScreen.status
+    const noteDetail = document.querySelector(".NoteDetail")
+    const mainBody = document.querySelector(".Main__body___browser-main-")
+    const sliderRight = document.querySelector(".Main__slider-right___browser-main-")
+    const slider = document.querySelector(".Main__slider___browser-main-")
```

Comment:
Those class names such as `.Main__slider___browser-main-` depends on the specification of webpack, so could you name any other one to them?